configure.ac

```diff
@@ -270,6 +270,8 @@ AC_CONFIG_FILES([
 	tests/zfs-tests/tests/functional/cli_root/zpool_trim/Makefile
 	tests/zfs-tests/tests/functional/cli_root/zpool_upgrade/Makefile
 	tests/zfs-tests/tests/functional/cli_root/zpool_upgrade/blockfiles/Makefile
+	tests/zfs-tests/tests/functional/cli_root/zpool_wait/Makefile
+	tests/zfs-tests/tests/functional/cli_root/zpool_wait/scan/Makefile
```

Comment:
Given that every test requires some kind of privilege and technically zpool wait requires none, I think the best course of action is to leave the already written tests in cli_root and add a new test in cli_user that verifies the command in all its forms can run as a regular user. I think it’s best to have the sudo type stuff be relegated to the framework so that the tests can remain ignorant of it. Sound reasonable?

Reply:
@jwk404's suggestion makes good sense to me.  In fact, we can't rely on being able to call `sudo` from the `cli_user` tests in order to execute a privileged commands since it may not be allow.  We recently got burned by this in another test case, see 4302698be16e88419b8b22194685372e012fa333

Reply:
That sounds good to me, I'll do that.